src/common/common-minicore.vcxproj

```diff
@@ -126,8 +126,8 @@
       <Optimization>Disabled</Optimization>
       <PreprocessorDefinitions>$(DefineConstants);WIN32;_CRT_SECURE_NO_DEPRECATE;_CRT_NONSTDC_NO_DEPRECATE;_WINSOCK_DEPRECATED_NO_WARNINGS;LIBCONFIG_STATIC;YY_USE_CONST;MINICORE;_DEBUG;_LIB;_ITERATOR_DEBUG_LEVEL=0;%(PreprocessorDefinitions)</PreprocessorDefinitions>
       <RuntimeLibrary>MultiThreadedDebug</RuntimeLibrary>
-      <AdditionalIncludeDirectories>$(SolutionDir)3rdparty\zlib\include\;$(SolutionDir)3rdparty\mysql\include\;$(SolutionDir)3rdparty\libconfig\;$(SolutionDir)3rdparty\yaml-cpp\include\</AdditionalIncludeDirectories>
-      <AdditionalIncludeDirectories>$(SolutionDir)3rdparty\zlib\include\;$(SolutionDir)3rdparty\libconfig\</AdditionalIncludeDirectories>
+      <AdditionalIncludeDirectories>$(SolutionDir)src;$(SolutionDir)3rdparty\zlib\include\;$(SolutionDir)3rdparty\mysql\include\;$(SolutionDir)3rdparty\libconfig\;$(SolutionDir)3rdparty\yaml-cpp\include\</AdditionalIncludeDirectories>
```

Comment:
This file looks strange (not your fault).
MySQL is definitely not a dependency of minicore. YAML-CPP also isnt.
Not sure about zlib and libconfig though.
My guess would be that the second line is correct and the first one should be removed.
This is also the line which MSVS uses at the moment. It seems to ignore the first line.
![grafik](https://user-images.githubusercontent.com/3517879/218267057-4a569fbd-4635-4795-bb12-582f42ddf55f.png)

Reply:
I didn't even notice, I just did a search replace

MSVS must take the latest value, the second overwrites the first.
I think minicore does has a dependency on libconfig, since we use the libconfig header in conf.hpp
same with zlib, in grfio